client/auth/login.jsx

```diff
@@ -167,14 +194,23 @@ module.exports = React.createClass( {
 					</FormFieldset>
 					<FormButtonsBar>
 						<FormButton disabled={ ! this.canSubmitForm() } onClick={ this.recordClickEvent( 'Sign in' ) } >
-							{ requires2fa ? this.translate( 'Verify' ) : this.translate( 'Sign in' ) }
+							{ requires2fa === 'code' ? this.translate( 'Verify' ) : this.translate( 'Sign in' ) }
 						</FormButton>
 					</FormButtonsBar>
 					{ ! requires2fa && <LostPassword /> }
 					{ errorMessage && <Notice text={ errorMessage } status={ errorLevel } showDismiss={ false } /> }
-					{ requires2fa && <AuthCodeButton username={ this.state.login } password={ this.state.password } /> }
+					{ requires2fa === 'push-verification' &&
+					<FormButtonsBar>
+						<FormButton ref="useAuthCode" onClick={ this.useAuthCode } >
+							{ this.translate( 'Verify with code instead' ) }
+						</FormButton>
+					</FormButtonsBar>
+					}
+					{ ( requires2fa === 'code' ) && <AuthCodeButton username={ this.state.login } password={ this.state.password } /> }
```

Comment:
Similar to the above comment, can we put the components on a new and indented line?